helpers/fetchApiData.ts

```diff
@@ -18,7 +18,7 @@ export async function fetchApiData<T>({
   method?: "GET" | "POST" | "PUT" | "DELETE" | "PATCH";
   headers?: HeadersInit;
   body?: object;
-  bearerToken: string;
+  bearerToken?: string;
```

Comment:
The endpoint used inside the repo pages doesn't require a bearer token, and since I used this function to use the API, I made it optional so non logged in users can still see the repo page. I can revert and manually do the fetch, but figured this is the easiest way to use the API.

Reply:
No, all good. I just needed some context is all. If it's optional now though, I'd skip adding the Authorization header. Something like this.

```diff
  const init: RequestInit = {
    method,
    headers: {
      ...headers,
      accept: "application/json",
      "Content-Type": "application/json",
-      Authorization: `Bearer ${bearerToken}`,
+.     ...(bearerToken ? { Authorization: `Bearer ${bearerToken}` } : {}),
    },
  };
```